modules/install/pages/install.adoc

```diff
@@ -1,71 +1,538 @@
-= Installing JBoss EAP
+== Red Hat JBoss Enterprise Application Platform 8 Installation Methods
 
-Brief blurb about how JBoss EAP offers multiple ways to install...
+You can install JBoss EAP 8.0 by using one of the following installation methods:
 
-== Pre-requisites
-
-Talk about what pre-requisites are needed:
+* *JBoss EAP using the Zip file*
```

Comment:
Remove JBoss EAP using prefix in all these bullets

Reply:
On first reading, zip file and archive look the same? is archive packaged as tar.gz?

Reply:
done the correction